apps/frontline-widgets/src/formBundle.js

```
@@ -1,268 +1,454 @@
+    w(e, { action: 'extraFormContent', html: t });
   });
-  var formSettings = window.erxesSettings.forms || [];
-  var iframesMapping = {};
-  var getMappingKey = (settings) => JSON.stringify({
-    form_id: settings.form_id,
-    channel_id: settings.channel_id
+  var k = window.erxesSettings.forms || [],
+    f = {},
+    v = (e) => JSON.stringify({ form_id: e.form_id, channel_id: e.channel_id }),
+    R = (e) =>
+      k.find((t) => t.channel_id === e.channel_id && t.form_id === e.form_id);
```

ESLint

[error] Unexpected var, use let or const instead.

(no-var)